api/src/main/java/io/strimzi/api/kafka/model/KafkaRebalanceSpec.java

```diff
@@ -41,6 +41,7 @@
     private int concurrentIntraBrokerPartitionMovements;
     private int concurrentLeaderMovements;
     private long replicationThrottle;
+    private String replicaMovementStrategies;
```

Comment:
do you think we could use a `List<String>` ? the order should be preserved when serialized/deserialized right? or am I wrong?

Reply:
The order should be preserved. This is like the goals list in the `cruiseControl.spec`.

Reply:
So isn't it much better having a YAML array as we have for goals instead of a string with comma separated values? Just my 2 cents, but no need to change if you feel it's better in the current way. I just found YAML array a better fit for this field.

Reply:
We don't have a YAML array for the goals config in the Kafka CR, we have a single comma separated string split over several lines:
```
cruiseControl:
    capacity:
      networkIn: 10000KB/s
      networkOut: 10000KB/s
    config:
      default.goals: >
           com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal,
           com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaCapacityGoal,
           com.linkedin.kafka.cruisecontrol.analyzer.goals.DiskCapacityGoal
```